libs/src/services/dataContracts/AudiusContracts.ts

```diff
@@ -72,6 +59,7 @@ export class AudiusContracts {
 
     this.RegistryClient = new RegistryClient(
       this.web3Manager,
+      // @ts-ignore
```

Comment:
ts doesnt convert json into ABI[] annoyingly, will follow up with an `as ABI[]` if we think that's more ideal